### PR TITLE
Document the new endpoints for read/create/update of access groups

### PR DESCRIPTION
**Pull request by ivaylomitrev**
This commit describes several new endpoints:
* GET/POST/PUT rms/api/public/noark5/v1/access-group/

### CODE REVIEW

File: version1/04 - Web services.md [hunk 1]
```diff
@@ -1287,6 +1287,276 @@ Content-type: application/json
...
+
+200 OK
+
+```text
+Content-type: application/json
+
+{
+    "groups": [
+        {
+            "id": integer,
```
Comment (vulkodava) [inline version1/04 - Web services.md]: Should this be long as per PR #45?

File: version1/04 - Web services.md [hunk 2]
```diff
@@ -1287,6 +1287,276 @@ Content-type: application/json
...
+
+**Response**
+
+200 OK
+
+```text
+Content-type: application/json
+
+{
+    "id": integer,
```
Comment (vulkodava) [inline version1/04 - Web services.md]: Probably the type here must be long instead.

File: version1/04 - Web services.md [hunk 3]
```diff
@@ -1287,6 +1287,276 @@ Content-type: application/json
...
+- **global permissions** (optional)
+  - the group global explicit permissions
+  - must contain only explicit permissions as defined in the "Access control" documentation
+- **service permissions** (optional)
+  - the group service permissions
+  - must contain only service permissions as defined in the "Access control" documentation
+
+**Response**
+
+200 OK
```
Comment (vulkodava) [inline version1/04 - Web services.md]: Should the response be 201 Created?

File: version1/04 - Web services.md [hunk 4]
```diff
@@ -1287,6 +1287,276 @@ Content-type: application/json
...
+
+```text
+Content-type: application/json
+
+{
+    "id": integer,
```
Comment (vulkodava) [inline version1/04 - Web services.md]: Here type too.

File: version1/04 - Web services.md [hunk 5]
```diff
@@ -1287,6 +1287,276 @@ Content-type: application/json
...
```
Comment (vulkodava) [inline version1/04 - Web services.md]: Type.

**Review by vulkodava — CHANGES_REQUESTED** [review]

File: version1/04 - Web services.md (hunk 1 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed with https://github.com/documaster/noark5-web-services/pull/47/commits/0bc6296e6cdab8f679ef5bf3a2e5c1b519134158.

File: version1/04 - Web services.md (hunk 2 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed with https://github.com/documaster/noark5-web-services/pull/47/commits/0bc6296e6cdab8f679ef5bf3a2e5c1b519134158.

File: version1/04 - Web services.md (hunk 4 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed with https://github.com/documaster/noark5-web-services/pull/47/commits/0bc6296e6cdab8f679ef5bf3a2e5c1b519134158.

File: version1/04 - Web services.md (hunk 5 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed with https://github.com/documaster/noark5-web-services/pull/47/commits/0bc6296e6cdab8f679ef5bf3a2e5c1b519134158.

I like how each subsequent comment became shorter and shorter :D

File: version1/04 - Web services.md (hunk 3 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed with https://github.com/documaster/noark5-web-services/pull/47/commits/f426db2322c123f1e647f27a0093544d62c5e9bc.

**Comment by ivaylomitrev** [comment]
Comments thus far have been addressed.

**Review by vulkodava — APPROVED** [review]

**Review by teodoradobreva — APPROVED** [review]

**Review by orlinDocumaster — APPROVED** [review]

File: version1/04 - Web services.md [hunk 6]
```diff
@@ -1287,6 +1287,276 @@ Content-type: application/json
...
+
+- **id**
+  - the ID of the group
+- **name**
+  - the name of the group
+- **description**
+  - the description of the group
+- **claims**
+  - the group claims
+- **global permissions**
```
Comment (plamenxpenchev) [inline version1/04 - Web services.md]: Inconsistent with the rest of the Details sections in the file. Should be globalPermissions across the endpoints. Similarly for service permissions.

**Review by plamenxpenchev — CHANGES_REQUESTED** [review]

File: version1/04 - Web services.md (hunk 6 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Good point! Fixed with https://github.com/documaster/noark5-web-services/pull/47/commits/adf97e97f75ed8e10906c634b1eb9f26df452fb1.

**Review by plamenxpenchev — APPROVED** [review]

**Review by teodoradobreva — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
Squashed and rebased in preparation for merging.

**Pull request merged**